.github/workflows/ci.yml

```diff
@@ -14,10 +14,10 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-        - 2.7
-        - 3.5
-        - 3.7
-        - 3.9
+        - '2.7'
+        - '3.5'
+        - '3.7'
+        - '3.10'
```

Comment:
I replaced 3.9 with 3.10 here as I assumed that since there was no 3.8 that after 3.7 the latest version would just get updated. Let me know if this assumption was wrong.